edx_sga/sga.py

```diff
@@ -57,69 +57,69 @@ class StaffGradedAssignmentXBlock(XBlock):
     icon_class = 'problem'
 
     display_name = String(
-        default='Staff Graded Assignment', scope=Scope.settings,
-        help="This name appears in the horizontal navigation at the top of "
-             "the page."
+        default= "Staff Graded Assignment", scope=Scope.settings,
+        help= _("This name appears in the horizontal navigation at the top of ") +
+              _("the page.")
     )
 
     weight = Float(
-        display_name="Problem Weight",
-        help=("Defines the number of points each problem is worth. "
-              "If the value is not set, the problem is worth the sum of the "
-              "option point values."),
+        display_name=_("Problem Weight"),
```

Comment:
Waqas this is an x block it doesn't need to strictly conform to our
standards.
On Mar 11, 2015 7:19 AM, "Waqas Khalid" <notifications@github.com> wrote:

> In edx_sga/sga.py
> <https://github.com/mitodl/edx-sga/pull/65#discussion_r26204096>:
>
> >      )
> >
> >      weight = Float(
> > -        display_name="Problem Weight",
> > -        help=("Defines the number of points each problem is worth. "
> > -              "If the value is not set, the problem is worth the sum of the "
> > -              "option point values."),
> > +        display_name=_("Problem Weight"),
>
> check the earlier comments space missing
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mitodl/edx-sga/pull/65/files#r26204096>.
>